### ISSUE TITLE
[Bug] Rectangle missing width and height? 

### ISSUE THREAD

**Issue by solewalker** [bjorn/tiled#1137]
One thing I noticed, if you select Rectangle (R) and click, a rectangle is placed with height and width to 0. Is it a bug or is it to specify a point? If it is the later case, then there should be a better visual for it when not selected. Otherwise it is exactly like a regular rectangle, no visual difference between the two.

![tiled-bug-1](https://cloud.githubusercontent.com/assets/2183770/10870374/95b2f6ce-80f3-11e5-8926-4482a293589e.gif)

**Comment by bjorn** [comment]
Yeah, this is to make point objects more visible and easier to work with, but I totally agree they need a better visualization because this is rather confusing.

**Comment by bjorn** [comment]
Another option to consider is to leave the rectangle and ellipse as dots when their size is 0,0 and to introduce a new "position/point" object type.

**Comment by solewalker** [comment]
I think position/point object would make more sense and if it's not too much work. But a different way to visualize a point/position is what needed the most.

**Comment by bjorn** [comment]
For reference, there is issue #661 about point/marker objects.

**Comment by zommerfelds** [comment]
I don't know if anyone relies on "default" rectangles, but IMO they shouldn't be allowed to be placed as it could break assumptions people make about rectangles :) So my vote goes to clicking should place a rectangle with an actual default width and height. I feel like this will be most useful UI for people, because if you place a rectangle you probably expect a valid rectangle shape, not a point or collapsed rectangle. Then people can easily edit to any size they want.
